django_comparison_dashboard/forms.py

```diff
@@ -431,6 +434,14 @@ class SankeyGraphFilterSet(FilterSet):
     }
 
     def __init__(self, data: dict | None = None, data_filter_set: DataFilterSet | None = None):
+        initial_data = {
+            "nodes": "default_node_choice",
+            "inflow": "default_inflow_choice",
+            "outflow": "default_outflow_choice",
+        }
+        if data:
+            # Merge initial_data with the incoming data
+            initial_data.update(data)
```

Comment:
Currently, `initial_data` is not used. Do you tried something here and forgot to delete it afterwards?
Otherwise, I would remove it.